iceoryx_hoofs/test/moduletests/test_container_uninitialized_array.cpp

```diff
@@ -100,6 +100,39 @@ TEST(UninitializedArrayTest, capacityIsCorrect)
     EXPECT_EQ(sut.capacity(), capacity);
 }
 
+TEST(UninitializedArrayTest, isNotCopyConstructible)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "abc31a08-77b2-4fd2-af14-3129bafda00c");
+
+    bool is_copy_constructible = std::is_copy_constructible<UninitializedArray<int, 31>>::value;
+    EXPECT_FALSE(is_copy_constructible);
+}
+
+TEST(UninitializedArrayTest, isNotCopyAssignable)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "42c31a08-77b2-4fd2-6914-3129869da00c");
+
+    bool is_copy_assignable = std::is_copy_assignable<UninitializedArray<int, 31>>::value;
```

Comment:
We could vary the capacity `31` from test to test. It should not depend on it, and I would not really check for multiple values, but it could be different from test to test to make it more "likely" (deliberately imprecise) to catch defects.